config/locales/en.yml

```diff
@@ -1616,7 +1629,7 @@ en:
         date_to_copy: Request all data held up until this date
       onwards: "onwards"
       update:
-        log_message_pages_changed: "%{location}, %{request_type}: pages changed from %{old_pages} to %{new_pages}"
+        log_message_pages_changed: "%{location} %{request_type}: pages changed from %{old_pages} to %{new_pages}"
```

Comment:
Should the comma be removed?

Reply:
For this and the above, if a data request is edited but there isnt yet a location set for the data request area, the case history will show as 

", All prison records: pages ..."

Could do something dynamic to write the comma if its worth keeping it

Reply:
I don't think you should be able to add a data request without a location so that's probably something that needs to change. On that basis I wouldn't remove the comma.